config/repo-config.csv

```diff
@@ -8,3 +8,5 @@ https://github.com/reposense/testrepo-Delta.git,add-binary-file,,,,,,,
 https://github.com/reposense/RepoSense.git,master,,,,,,,
 https://github.com/reposense/testrepo-Empty.git,master,,,,,,,
 ftp://github.com/reposense/RepoSense.git,master,,,,,,,
+https://bitbucket.org/skyblaise/testrepo-bitbucket.git,master,,,,,,,
+https://gitlab.com/reposense/testrepo-gitlab.git,main,,,,,,,
```

Comment:
Does this repo exist?

Reply:
I created it and made it public. You cannot access or view it?

Reply:
The bitbucket one is accessible. The gitlab one is not accessible even while I'm logged in. I can't tell if the repo exists though your profile does not have any visible repos either

Reply:
After some checking, the visibility is somehow set to `private` and I'm unable to change it to `public`, which is weird. Seems to be some problem from GitLab's side. Anyways, I will remove the 2 repos as you mentioned so it should be a problem.